packages/charts/src/renderers/area/area-renderer.ts

```diff
@@ -103,20 +103,6 @@ export class AreaRenderer extends XYRenderer<IAreaAccessors> {
         path.attr("d", areaFunc(validatedData) ?? "");
     }
 
-    /** See {@link Renderer#getDataPointIndex} */
```

Comment:
I think we can remove the getDataPointIndex override from the LineRenderer as well.